xml/vt_best_practices.xml

```diff
@@ -3879,14 +3876,14 @@ VGA trouble debug
   </sect2>
  </sect1>
 
-<sect1 xml:id="vt.best.Xen_pv_fv">
+<sect1 xml:id="vt.best.xen_pv_fv">
   <title>&xen;: Moving from PV to FV</title>
   <para>
    This chapter explains how to convert a &xen; para-virtual machine into a
    &xen; fully virtualized machine.
   </para>
   <para>
-   First you need to change to the <emphasis>-default</emphasis> kernel, if not
+   First you need to change to the <emphasis>-default</emphasis> kernel; if not
```

Comment:
First you need to change to the default Kernel (package <package>kernel-default</papckage>); if not
￼already installed, you need to install it while in PV mode.

_Note: No "must", it should rather be "need to" (according to the Styleguide)._